tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot

```diff
@@ -114,6 +114,7 @@ Move To Installed Operator Page Tab in Openshift
     Installed Operators Should Be Open
     Run Keyword If  "${namespace}" == "None"   Select Project By Name  All Projects
     ...         ELSE   Select Project By Name   ${namespace}
+    Sleep   1s
```

Comment:
Is it possible to replace this Sleep for Wait Until Element Is Visible?
What about adding the Sleep (or the Wait Until ...) inside `Select Project By Name `?

Reply:
yeah, here the thing was that the element was apparently visible, but it could not interact with, so I found that a little sleep worked. The keyword `Click On Searched Operator` already waits for the element to be visible for 150seconds. It looked like a random error happening.
I agree with replacing the sleep, but I want to see how it works over few run in our CI. After that I will try to improve it